CONTRIBUTING.md

```diff
@@ -154,5 +154,6 @@ The README of the CLI repository can be found [here](https://github.com/hasura/g
 This document is available in the following translations:
 
 - [French 🇫🇷](translations/CONTRIBUTING.french.md)
+- [Indonesian :indonesia:](translations/CONTRIBUTING.indonesian.md)
```

Comment:
This link won't work since you've named the translation file `translations/CONTRIBUTING.indonesia.md`. 
```suggestion
- [Indonesian :indonesia:](translations/CONTRIBUTING.indonesia.md)
```